### DISCUSSION TITLE
Metabolizing enzyme for an ADC model

### DISCUSSION THREAD

**Discussion by he-kadry** [Open-Systems-Pharmacology/Forum#1438] [Questions & Problems]
I am working with an ADC model and I added two metabolizing enzymes for the ADC to produce two payloads(toxophore) and I selected the two compounds that I created as the produced metabolites. Is there a way to simulate the release the of the payloads? Currently, I am able to simulate only the ADC profile.

**Comment by prvmalik** [comment]
If you set it up correctly, then define settings > select your small molecule observers > run should reveal them.

Note that you have to adjust the endothelial permeability for the small molecules to about 100 if I recall correctly in order for them to work properly as a small molecule within the framework of the large molecule model

**Reply by he-kadry** [reply]
So, I added a metabolizing enzyme within my ADC compound ADME properties and I selected the payload, that I created as a separate compound, as the metabolite. Then, I created a simulation and included both compounds and selected a large molecule model for my simulation with dose only for the ADC. I run the simulation, but only the ADC profile is showing? What do you think may be the error?

**Reply by prvmalik** [reply]
did you select the observer that shows the small molecule compound

**Reply by prvmalik** [reply]
is the enzyme intracellular? no ADC will make it there.

**Reply by he-kadry** [reply]
I didn't find a way to specify if the enzyme is intra or extracellular!

**Reply by prvmalik** [reply]
Setting the enzyme as present in plasma is done below the reference concentration in the "expression" tab. 

But, based on this discussion I would advise extreme caution in using or interpreting the results. An ADC model is presently the most complex form of PBPK available. The ADC can be degraded in the plasma (deconjugation) releasing payload there, and can also be degraded in the endosomes, where it also releases payload. The payload then passively diffuses out of endosomes and into plasma. To capture these processes you will need a deconjugation enzyme in plasma, and will need to replace the normal degradation process of the ADC (Kdeg) with a degradation enzyme that produces the metabolite in the endosomal space. Now we also will need to shift the model to Mobi because the stoichiometry of the reaction needs to be increased according to the drug-to-antibody ratio (DAR). Most ADCs have a DAR around 4. With the basic setup, only one molecule of payload would be released from the ADC when it is degraded.

And that does not consider any TMDD.

**Comment by StephanSchaller** [comment]
Dear @he-kadry, I guess you may have forgotten to actually specify the metabolite in the definition of the metabolic process and when configuring the simulation:
Molecule BB in the Tab "ADME" when creating the metabolic process:
![image](https://user-images.githubusercontent.com/25477643/212565927-83f6fd8d-48a9-4d37-b655-daae98fc9517.png)

And when creating the simulation in this tab:
![image](https://user-images.githubusercontent.com/25477643/212566510-3f3ef112-f11a-4357-bfcd-95e03a6f373d.png)

**Reply by he-kadry** [reply]
I created the metabolite compound before adding the enzyme and I added the small molecule where you showed as a metabolite. Now, I can't find a place in simulation to get the metabolite profile!! can I specify the enzyme to plasma or how to create a deconjugation process for the ADC?

**Comment by StephanSchaller** [comment]
@he-kadry , I can recommend going through some courses for PK-Sim/MoBi to learn how to set up an ADC model from scratch.
The course version, for now, neglects the release in the endosome. But gives you the skills to implement it.

If you are in academia, you can register for free here: https://learn-esqlabs.talentlms.com/ --> Antibody Ther. in Onc.
From industry, you can purchase access here: https://esqlabs.shop/products/antibody-therapy-in-oncology-2
